Purchases/Caching/RCDeviceCache.m

```diff
@@ -313,6 +314,14 @@ - (void)storeAttributesForAllUsers:(NSMutableDictionary<NSString *, NSDictionary
     return attributes;
 }
 
++ (RCSubscriberAttribute *)newAttributeWithDictionary:(NSDictionary<NSString *, NSObject *> *)dictionary {
```

Comment:
see #577, I couldn't migrate the API fully due to Swift compiler bugs with objc bridging

Reply:
we might have to rewrite some of the related stuff during the migration, then. That's annoying, but I think most things that are affected are probably internal

Reply:
Yup, 😢